site/src/pages/docs/advanced-metrics.mdx

```diff
@@ -48,6 +48,16 @@ Server.builder()
       .build();
 ```
 
+If you use gRPC services with armeria-spring-boot, you can collect the metrics
```

Comment:
How about just:
```
You can provide <type://GrpcMeterIdPrefixFunction> as a bean if you are using a Spring integration module:

@Bean
public MeterIdPrefixFunction myMeterIdPrefixFunction() {
    return GrpcMeterIdPrefixFunction.of("grpc.service");
}
```
because this paragraph is an extension of the previous one?